src/Microsoft.AspNetCore.Sockets.Client.Http/Internal/ServerSentEventsTransport.cs

```diff
@@ -92,6 +114,7 @@ public Task StartAsync(Uri url, IDuplexPipe application, TransferFormat transfer
                 return;
             }
 
+            using (response)
```

Comment:
While the response is disposed if we reach here, an error thrown above at `response.EnsureSuccessStatusCode();` won't dispose the response. Connection could be left hanging because of `HttpCompletionOption.ResponseHeadersRead`.

Reply:
good catch, I blame @mikaelm12. Fixing it.